test_ee_raster.ipynb

```diff
@@ -12,6 +12,783 @@
     },
```

Comment:
<p>Is there a reason not to move this import to the top of the file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf_common/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Ack. Imports are now on the top of the file.